packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartAddressFormUpdate/useCounterpartAddressFormUpdate.tsx

```diff
@@ -27,6 +27,8 @@ export function useCounterpartAddressFormUpdate({
   counterpartId,
   addressId,
   onUpdate,
+  // ToDo: check why we are not using onCancel here, but passing it as prop from parent component
```

Comment:
You can remove it because it is just object destructuring

Reply:
If we remove this `onCancel`, it would also be logical to eliminate the interface for `onCancel` as well. However, doing so could lead to further complications:

<img width="1092" alt="Screenshot 2024-07-31 at 22 07 52" src="https://github.com/user-attachments/assets/ee67b424-fbb2-44ce-961d-e25d4ce4d771">

Should we consider removing the onClick functionality entirely? It seems we may have identified a potential flaw in the application